apps/gp2-frontend/src/onboarding/__tests__/Background.test.tsx

```diff
@@ -97,9 +97,7 @@ describe('Background', () => {
   });
 
   it('saves the keywords modal', async () => {
```

Comment:
I don't know if the modal will change to Tag modal, but if so, the test name should be updated as well